### DISCUSSION TITLE
don't work

### DISCUSSION THREAD

**Discussion by Prist92** [community/community#131723] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Ребята, у меня на телефоне приложение не может воспроизвести видео. Оно бесконечно загружается, а потом телефон пишет, что Ошибка сети, хотя интернет есть. И раньше все было хорошо. Телефон Redmi note 12 pro 5g. А вот на планшете все работает. Странно.

**Comment by TheGiraffe3** [comment]
Видео? Вы имели в виду видеоигру? И еще бесконечное количество вопросов: что это за игра?

**Comment by Prist92** [comment]
Нет. Я наверное что-то не так сделал. Но я имел в виду программу NewPipe

ср, 10 июл. 2024 г., 21:04 Loymdayddaud ***@***.***>:

**Reply by TheGiraffe3** [reply]
Хорошо. Рад, что у вас все заработало.

**Reply by Prist92** [reply]
Не. Пока не заработало. ))
Программа не очень хочет это делать ))
Хотя раньше работала как прекрасно.
Но я надеюсь на лучшее.

**Reply by Prist92** [reply]
Всё. Теперь и на планшете не работает ((

**Reply by Prist92** [reply]
О! Обновление 027.1 Заработало!!!!! Спасибо!!!

**Reply by Prist92** [reply]
Вы большие молодцы
